tests/Mono.Android-Tests/Android.Graphics/NinePatchTests.cs

```diff
@@ -25,6 +25,7 @@ public class NinePatchTests
 		};
 
 		[Test, TestCaseSource (nameof (NinePatchDrawables))]
+		[DynamicDependency (DynamicallyAccessedMemberTypes.All, typeof (NinePatchDrawable))]
```

Comment:
This test was failing, where the only usage of `NinePatchDrawable` in C# was:
```csharp
Assert.IsNotNull (d as NinePatchDrawable);
```
And so the type `NinePatchDrawable` existed, but with everything trimmed away.